pyxdameraulevenshtein/pyxdameraulevenshtein.pyx

```diff
@@ -23,7 +23,7 @@
 
 from cpython.version cimport PY_MAJOR_VERSION
 from libc.stdlib cimport calloc, free
-import numpy as np
```

Comment:
Why did you remove this import? The [Cython docs](http://docs.cython.org/en/latest/src/tutorial/numpy.html#adding-types) both `import` and `cimport` numpy.